modules/alloy/src/main/ts/ephox/alloy/behaviour/reflecting/ReflectingState.ts

```diff
@@ -1,20 +1,18 @@
-import { Cell, Fun, Optional } from '@ephox/katamari';
+import { Cell, Optional } from '@ephox/katamari';
 
 import { ReflectingState } from './ReflectingTypes';
 
 const init = <S>(): ReflectingState<S> => {
-  const cell: Cell<Optional<S>> = Cell(Optional.none<S>());
+  const cell = Cell(Optional.none<S>());
```

Comment:
Why did this one not become a Singleton?

Reply:
Because the input data is `Optional` not an actual value, so `Cell` is a more appropriate data structure for this.